db-experiment/Vagrantfile

```diff
@@ -0,0 +1,28 @@
+# -*- mode: ruby -*-
+
+box = "generic/ubuntu1804"
+ram = "65536"
+hostname = "wpia-db-experiment"
+mac = "00:15:5d:1a:84:10"
+cpus = 2
+
+Vagrant.configure(2) do |config|
+  config.vm.box = box
+  config.vm.network "public_network", bridge: "External Switch", auto_config: true
+  config.vm.provider :hyperv do |hyperv|
+    hyperv.mac = mac
+    hyperv.vmname = hostname
+    hyperv.cpus = cpus
+    hyperv.maxmemory = ram
+    hyperv.memory = "65536"
```

Comment:
Looks like this could go to 256GB I think?

Reply:
Upped that to 256